cliquet/resource.py

```diff
@@ -110,10 +110,12 @@ def collection_get(self):
         records, total_records, next_page = self.get_records()
 
         headers = self.request.response.headers
-        headers['Total-Records'] = ('%s' % total_records).encode('utf-8')
+        headers['Total-Records'] = ('%s' % total_records)
 
         if next_page:
-            headers['Next-Page'] = next_page.encode('utf-8')
+            if next_page.__class__ is not str:
+                next_page = next_page.encode('utf-8')
```

Comment:
This reminds me:

https://github.com/elastic/elasticsearch-py/issues/113

https://github.com/elastic/elasticsearch-py/commit/18d6b41e8299d3e62bfac2a8c2f8ceb87be24f45

https://github.com/elastic/elasticsearch-py/commit/5fec49e2618ccab706ea360b18d57420478cd265